plugins/Login/PasswordResetter.php

```diff
@@ -104,6 +104,11 @@ class PasswordResetter
      */
     private $emailFromAddress;
 
+    /**
+     * Hash algorithm used to create the password reset token.
+     */
+    const TOKEN_HASH_ALGO = 'sha3-512';
```

Comment:
fyi would maybe need to check if this hash algo is available and if not fall back to a different one

Reply:
According to [PHP hash_algos()](https://www.php.net/manual/en/function.hash-algos.php), support for  sha3-512 was added in PHP7.1, and the minimum requirement for Matomo is PHP7.2.